### ISSUE TITLE
More height variation in some levels

### ISSUE THREAD

**Issue by E-the-Z** [studio-minus/madness-interactive-reloaded#249]
Currently the only directions you can shoot are on both sides, so I think it would be nice if some levels had a second floor or catwalk that you could climb up to and shoot down from. Enemies could also spawn on top of it.

**Comment by Orsoniks** [comment]
I'm pretty sure thats impossible with the way this game works at its core

**Comment by dch-GH** [comment]
> I'm pretty sure thats impossible with the way this game works at its core

Yep, we use a "floor line" of which there can only be one and it cannot wrap around itself in anyway.
